lib/view_component/config.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+require "view_component/deprecation"
+
+module ViewComponent
+  class Config < ActiveSupport::InheritableOptions
+    class << self
+      alias default new
```

Comment:
Sure! I mostly put it in just in case `new`'s output ever changes, since they _could_ diverge in theory — it's a convenience that new configs have the defaults.